src/ck-core/ckmigratable.h

```diff
@@ -123,14 +123,6 @@ class CkMigratable : public Chare {
   int prefetchObjID; //From CooRegisterObject
   bool isInCore; //If true, the object is present in memory
 #endif
-
-#if CMK_FAULT_EVAC
-private:
-  bool asyncEvacuate;
-  void AsyncEvacuate(bool set){myRec->AsyncEvacuate(set);asyncEvacuate = set;};
-public:
-  bool isAsyncEvacuate(){return asyncEvacuate;};
-#endif
```

Comment:
Would the removal of the AsyncEvacuate method in this PR break compilation with `-DCMK_FAULT_EVAC=1`? TCharm and AMPI both call `this->AsyncEvacuate` from chare arrays.

Reply:
I was under the impression that FAULT_EVAC is no longer being supported.

Reply:
If it's not then all CMK_FAULT_EVAC blocks should be removed in a separate PR preceding this one. We should discuss this in Core today.